src/swap/hooks/useFromTo.ts

```diff
@@ -28,6 +36,9 @@ export const useFromTo = (address?: Address) => {
     loading: fromLoading,
     setLoading: setFromLoading,
     error: fromTokenBalanceError,
+    refetch: async () => {
```

Comment:
Not a fan of this, as it creates weird bad practices.

I want us to think more into the Life Cycle Status mindset.

Like what status we are changing, and that will trigger an action.

Reply:
PR was incomplete when you reviewed - please check the updated diffs

to clarify, do you mean returning the entire `tokenResponse` instead of just the `refetch` attribute of the `tokenResponse` object here? I doubt we'll need the other fields

Reply:
either `refetch` or the entire object has to be returned here for us to be able to refetch the token balances from the `SwapProvider` lifecycle status hook